app/src/AppRoutes.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Suspense } from 'react';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Loading from 'components/common/Loading';
+import { Layout } from 'components/layout';
+
+const LazyAuthPage = React.lazy(() => import('components/auth/AuthPage'));
+const LazyLoopPage = React.lazy(() => import('components/loop/LoopPage'));
+const LazyHistoryPage = React.lazy(() => import('components/history/HistoryPage'));
+const LazyPoolPage = React.lazy(() => import('components/pool/PoolPage'));
+const LazySettingsPage = React.lazy(() => import('components/settings/SettingsPage'));
+const LazyConnectPage = React.lazy(() => import('components/connect/ConnectPage'));
+
+const AppRoutes: React.FC = () => {
+  return (
+    <Suspense fallback={<Loading delay={500} />}>
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<LazyAuthPage />} />
+          <Route>
+            <Route
+              path="loop"
+              element={
+                <Layout>
```

Comment:
Can't have `Layout` as child of `Routes` so wrapped lazy loaded components with it.
Optionally, I can have `Layout` in the components itself instead of here.